legacy/firmware/layout2.c

```
@@ -373,7 +371,7 @@ void refreshUsbConnectTips(void) {
 void layoutStatusLogoEx(bool fresh) {
```

Clang (14.0.6)

[warning] parameter 'fresh' is unused

(misc-unused-parameters)